src/foundry/client/pixi/placeable.d.ts

```diff
@@ -245,6 +278,14 @@ declare global {
      */
     protected _onRelease(options?: PlaceableObject.ReleaseOptions): void;
 
+    /**
+     * Clone the placeable object, returning a new object with identical attributes.
+     * The returned object is non-interactive, and has no assigned ID.
+     * If you plan to use it permanently you should call the create method.
+     * @returns A new object with identical data
+     */
+    clone(): ThisType<PlaceableObject>;
```

Comment:
I think you're misusing `ThisType` here. Is the idea that `clone` returns `this`? In which case do `this`. Is the idea that it's a temporary document? Is the idea that it returns a new `PlaceableObject`? Then return `PlaceableObject`. I guess I'm just unsure what's going on here.